pkg/fixtures/dackbox.go

```diff
@@ -0,0 +1,1162 @@
+	return &storage.EmbeddedNodeScanComponent{
+		Name:    "downtown-london",
+		Version: "1s.2-3",
+		Vulns: []*storage.EmbeddedVulnerability{
+			GetEmbeddedNodeCVE_3456_0004(),
+			GetEmbeddedNodeCVE_3456_0005(),
+		},
+		Vulnerabilities: nil,
+		Priority:        0,
+		SetTopCvss:      &storage.EmbeddedNodeScanComponent_TopCvss{TopCvss: 7.5},
+		RiskScore:       0,
+	}
+}
+
+// GetEmbeddedNodeComponent_2_4 provides a pseudo-realistic node component for dackbox integration testing.
+func GetEmbeddedNodeComponent_2_4() *storage.EmbeddedNodeScanComponent {
+	return &storage.EmbeddedNodeScanComponent{
+		Name:            "dr-jekyll-medecine-practice",
+		Version:         "2.4",
+		Vulns:           []*storage.EmbeddedVulnerability{},
+		Vulnerabilities: nil,
+		Priority:        0,
+		SetTopCvss:      &storage.EmbeddedNodeScanComponent_TopCvss{TopCvss: 0.0},
+		RiskScore:       0,
+	}
+}
+
+// GetEmbeddedNodeComponent_2_5 provides a pseudo-realistic node component for dackbox integration testing.
+func GetEmbeddedNodeComponent_2_5() *storage.EmbeddedNodeScanComponent {
+	return &storage.EmbeddedNodeScanComponent{
+		Name:    "mr-hyde-secret-entrance",
+		Version: "2.5",
+		Vulns: []*storage.EmbeddedVulnerability{
+			GetEmbeddedNodeCVE_4567_0002(),
+			GetEmbeddedNodeCVE_2345_0006(),
+			GetEmbeddedNodeCVE_2345_0007(),
+		},
+		Vulnerabilities: nil,
+		Priority:        0,
+		SetTopCvss:      &storage.EmbeddedNodeScanComponent_TopCvss{TopCvss: 7.8},
+		RiskScore:       0,
+	}
+}
+
+// GetScopedNode_1 provides a pseudo-realistic node with scoping information matching the input for
+// dackbox integration testing.
+func GetScopedNode_1(nodeID string, clusterID string) *storage.Node {
+	node := &storage.Node{
+		Id:                      nodeID,
+		Name:                    "sherlock-holmes",
+		Taints:                  nil,
+		ClusterId:               clusterID,
+		ClusterName:             "test-cluster",
+		Labels:                  nil,
+		Annotations:             nil,
+		JoinedAt:                &types.Timestamp{Seconds: 1643789433},
+		InternalIpAddresses:     nil,
+		ExternalIpAddresses:     nil,
+		ContainerRuntimeVersion: "",
+		ContainerRuntime: &storage.ContainerRuntimeInfo{
+			Type:    storage.ContainerRuntime_DOCKER_CONTAINER_RUNTIME,
+			Version: "20.10.10",
+		},
+		KernelVersion:    "",
+		OperatingSystem:  "Docker Desktop",
+		OsImage:          "",
+		KubeletVersion:   "",
+		KubeProxyVersion: "",
+		LastUpdated:      nil,
+		K8SUpdated:       nil,
+		Scan: &storage.NodeScan{
+			ScanTime:        &types.Timestamp{Seconds: 1654154292, Nanos: 870002400},
+			OperatingSystem: "Linux",
+			Components: []*storage.EmbeddedNodeScanComponent{
+				GetEmbeddedNodeComponent_1_1(),
+				GetEmbeddedNodeComponent_1_2(),
+				GetEmbeddedNodeComponent_1s2_3(),
+			},
+			Notes: nil,
+		},
+		SetComponents: &storage.Node_Components{Components: 3},
+		SetCves:       &storage.Node_Cves{Cves: 5},
+		SetFixable:    &storage.Node_FixableCves{FixableCves: 2},
```

Comment:
I guess same question here as above: in the current mock data, how is a node CVE classified as fixable or not fixable? I'd expect the `storage.EmbeddedVulnerability.SetFixedBy` to be it?